README.tpl

```diff
@@ -1,42 +0,0 @@
-**_Please be aware that version `v0.4.0` of rusty-cheddar will be based around the more general
```

Comment:
Yeah, and it's hard for the tool too see where to stick the generated docs in a checked-in README.md. Anyway, I've moved the changes into the source comments and template file, then regenerated the README. Better now?